src/main/java/uk/gov/hmcts/reform/em/hrs/dto/RecordingFilenameDto.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.em.hrs.dto;
```

Comment:
do we need two separate DTOs for hearingRecording and its name?

Reply:
The endpoint we're currently exposing only returns the names of the hearing recording files we've already ingested and the ones in progress.